### PR TITLE
Greimorian queen bugfixes

### PR DESCRIPTION
**Pull request by hazelrat**
- Greimorian queens now have functional rest and dead sprites.
  - Greimorian queens no longer continue to regenerate after being killed.
  - Greimorian queens now appear closer to the centre of the occupied tile while facing west or east.

For context, this is what the east/west sprites currently look like.

image

And this is how they look with this PR.

image

### CODE REVIEW

**Comment by hazelrat** [comment]
!bugfix

**Comment by hazelrat** [comment]
!review

File: code/modules/mob/living/simple_animal/hostile/spider_queen.dm [hunk 1]
```diff
@@ -127,4 +131,5 @@
 
 /mob/living/simple_animal/hostile/giant_spider/nurse/spider_queen/Life(seconds_per_tick, times_fired)
 	..()
-	adjustBruteLoss(-3)
+	if(stat != DEAD)
```
Comment (FluffyGhoster) [inline code/modules/mob/living/simple_animal/hostile/spider_queen.dm:134]: Make this a guard statement like in `/mob/living/simple_animal/Life(seconds_per_tick, times_fired)`

Comment (hazelrat) [reply code/modules/mob/living/simple_animal/hostile/spider_queen.dm]: Implemented.

**Review by FluffyGhoster — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**